_emails/us/nebraska/omaha.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Omaha, Nebrasks
+date: 2020-06-07 19:17:00 Z
+permalink: "/omaha"
+name: Letter to the Mayor and City Council
+city: Omaha
+state: NE
+recipients:
+- Pete.Festersen@cityofomaha.org
+- Ben.Gray@cityofomaha.org
+- Chris.Jerram@cityofomaha.org
+- Vinny.Palermo@cityofomaha.org
+- Rich.Pahls@cityofomaha.org
+- Brinker.Harding@cityofomaha.org
+- Aimee.Melton@cityofomaha.org
+- carrie.murphy@cityofomaha.org
+- hotline@cityofomaha.org
+subject: 
+body: |
+  Dear Mayor Jean Stothert and Members of City Council,
+
+  My name is [YOUR NAME] and I am a resident of [NEIGHBORHOOD/BOROUGH/CITY]. First, I demand justice for James Scurlock who was killed by Jake Gardner on May 30th, 2020. Jake Gardner has been released from police custody and has not been charged. This case must be officially reopened. Second, I urge you to begin defunding and demilitarizing the Omaha Police Department.
+
+  The 2020 budget for OPD is outrageously high and unwarranted. Omaha’s general budget for 2020 was $419.6 million and the OPD currently takes up nearly 40% of that budget, or $159.5 million. Since 2016, the OPD’s budget has increased 22% but the city’s clearance rates still remain relatively stagnant.
+
+  In 2019, OPD’s portion of the budget was at 38% while Public Works, Parks, and Libraries made up just 5.7%, 5.5%, and 3.3% of the budget respectively. The underfunding of schools, infrastructure, parks, and libraries is causing much of Omaha’s suffering. The aforementioned underfunding of public services paired with the overfunding of the police shows me as your constituent that you are merely putting Band-Aids over bullet holes.
+
+  As Omaha residents, I therefore demand that you take immediate action to do the following:
+
+      Reduce Omaha’s allocation from the General Fund by 50% (roughly $80 million)
```

Comment:
For a bullet list, do you need extra space between these items? or does YAML make this be a bullet list because they are indented?

Reply:
They don't actually appear as a bullet list once opened into an email client, so while I can't guess as to the original reason it was written up that way, I do think it helps it read better once there: 
![image](https://user-images.githubusercontent.com/12093125/84094217-48515880-a9ca-11ea-86ad-81331aba162e.png)
![image](https://user-images.githubusercontent.com/12093125/84094283-78006080-a9ca-11ea-842e-4f4963f29587.png)

but looking at this again I did notice an odd plural, "as Omaha residents, I", so I'll correct that